<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_user</class>
 <widget class="QDialog" name="add_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add user</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="root_password">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>240</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeated_password">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>109</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Root password</string>
   </property>
  </widget>
  <widget class="QLabel" name="captured_image">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>8</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_img_button">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>336</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_text_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>284</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create user</string>
   </property>
  </widget>
  <widget class="QLabel" name="debug_info">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>488</y>
     <width>745</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="english_alphabet">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>380</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Only english alphabet letters allowed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
